src/applications/personalization/account/components/AccountMain.jsx

```diff
@@ -70,11 +52,6 @@ class AccountMain extends React.Component {
             </p>
           </div>
         }
-        onCloseAlert={this.dismissMVIError}
```

Comment:
since the close button is being removed from the MVI alert, I'm also removing all logic code related to handling closing the alert.